tests/unit/protocols/input/ec2.json

```diff
@@ -34,7 +34,7 @@
           "Bar": "val2"
         },
         "serialized": {
-          "uri": "/",
+          "uri": "",
```

Comment:
but if this url will be non-empty then these tests will fail...
because test_protocols uses default requst object from library. and they fail without this change because I changed default path_url there from '/' to ''.